res/database/Default/user/therob.xml

```diff
@@ -3193,28 +3306,39 @@
 				<en>%NAME1% solves fast-paced cases in his cozy neighborhood. Chases are guaranteed.</en>
 				<pl>%NAME1% rozwiązuje szybko sprawy w swojej przytulnej okolicy. Pościgi są gwarantowane.</pl>
 			</description>
-
 			<children>
 				<scripttemplate index="0" product="2" licence_type="2" guid="TheRob-script-ser-act-DerFahnder-Ep1">
 					<title>
 						<de>TVTs Pilot</de>
-						<en>TVTs Pilot</en>
+						<en>TVT's Pilot</en>
 						<pl>TVTs Pilot</pl>
 					</title>
+					<description>
+						<de>Ein verschwundener Hund entpuppt sich als Teil eines Schmugglernetzwerks.</de>
```

Comment:
@pauldim: once merged, Polish descriptions could be added as well

Reply:
I've added the Polish versions via DeepL here too for now. @pauldim can you review? I guess for now and in the future it's better for discoverability to look for added / updated ´<pl>` tags rather than look for missing ones.

If we get more contributors for other languages, I guess this workflow won't be possible anymore, and I guess we can't centrally maintain all language versions.